src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public final class FigureSupplier {
+    private final Random picker = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        final int figureCount = 5;
+        final int startNum = 1;
+        final int limitNum = 50;
+
+        return switch (picker.nextInt(figureCount)) {
+            case 0 -> new Circle(colorSupplier.getRandomColor(),
```

Comment:
same for other figures

```suggestion
            case 0 -> new Circle(randomColor,
```